### PR TITLE
Codebuds mattermost publication bundle recipe

### PR DESCRIPTION
**Pull request by codebuds33**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by codebuds33** [comment]
Hello,

Sorry if I waste your time. This is the first time we try to add a recipe to a Bundle. Travis CI failed for symfony 3 and 4 but works for symfony5 which is fine as that is the only version we want this bundle for.

How can I specify this so that the pull request passes ?

Thanks very much for any reply :)

File: codebuds/mattermost-publication-bundle/0.3/config/packages/mattermost_publication.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+mattermost_publication:
+    webhook_url: '%env(MATTERMOST_WEBHOOK_URL)%'
+    channel: null
```
Comment (Nyholm) [inline codebuds/mattermost-publication-bundle/0.3/config/packages/mattermost_publication.yaml:3]: You dont need all these null values. 

You could add a link to your bundles documentation instead.

Comment (DennisdeBest) [reply codebuds/mattermost-publication-bundle/0.3/config/packages/mattermost_publication.yaml:3]: Hey, thanks for checking this out. I will do that.

Comment (codebuds33) [reply codebuds/mattermost-publication-bundle/0.3/config/packages/mattermost_publication.yaml]: Ok I have just pushed the changes

**Comment by codebuds33** [comment]
@Nyholm Thanks a lot ;)

**Pull request merged**